phpBB/docs/events.md

```diff
@@ -1072,6 +1072,20 @@ navbar_header_logged_out_content
 * Since: 3.1.0-RC1
 * Purpose: Add text and HTML in place of the username when not logged in.
 
+navbar_header_user_profile_append
```

Comment:
Actually it should be after `navbar_header_q` and before `navbar_header_username`?